### PR TITLE
Added option to prevent Prefix being added to output path

### PR DESCRIPTION
**Pull request by MrMikeJJ**
Pull request for https://github.com/mganss/XmlSchemaClassGenerator/issues/300

### CODE REVIEW

**Comment by MrMikeJJ** [comment]
For defect #300

**Comment by MrMikeJJ** [comment]
thinking about it, using String.Replace could cause problems if a short Prefix was used. A substring would be better. I will update if this Pull Request is something you would consider.

File: XmlSchemaClassGenerator.Console/Program.cs [hunk 1]
```diff
@@ -87,6 +88,7 @@ static void Main(string[] args)
                 { "r|order", "emit order for all class members stored as XML element", v => emitOrder = v != null },
                 { "c|pcl", "PCL compatible output", v => pclCompatible = v != null },
                 { "p|prefix=", "the {PREFIX} to prepend to auto-generated namespace names", v => namespacePrefix = v },
+                { "po|prefixOnOutput", "include the prefix on the path (default is enabled)", v => addPrefixToOutputPath = v != null },
```
Comment (mganss) [inline XmlSchemaClassGenerator.Console/Program.cs:91]: IMO the phrasing can be improved here. Perhaps: `"po|prefixPath", "include namespace prefix in output paths (default is enabled)"`

**Comment by mganss** [comment]
Thank you for this PR.

If you use `Substring()`/range you also need to make sure that the string actually starts with the prefix. The prefix is only used for autogenerated namespaces, not explicit namespaces.

Please also add unit tests.

**Comment by MrMikeJJ** [comment]
> The prefix is only used for autogenerated namespaces, not explicit namespaces.

Yes, this causes issues because of that so will close this. I am getting different results running the original version through a library vs the exe. Currently investigating. (when ran through the it always seems to get added).

**Pull request closed without merging**